doc/api/util.md

```diff
@@ -1393,6 +1393,30 @@ added: v10.0.0
 
 Returns `true` if the value is a native `External` value.
```

Comment:
Nit: it *might* be good to include an explanation in here about what an `External` value is but that can be done separately.

Reply:
@jasnell Thanks. Created a separate PR https://github.com/nodejs/node/pull/31214.